src/test/suite/fixtures/recorded/selectionTypes/takeEveryCarBlock.yml

```diff
@@ -0,0 +1,51 @@
+languageId: plaintext
+command:
+  spokenForm: take every car block
+  version: 3
+  targets:
+    - type: primitive
+      modifiers:
+        - type: everyScope
+          scopeType: {type: character}
+        - type: containingScope
+          scopeType: {type: paragraph}
+  usePrePhraseSnapshot: true
+  action: {name: setSelection}
+initialState:
+  documentContents: |-
+    aaa
+    bbb
+    ccc
+  selections:
+    - anchor: {line: 0, character: 0}
+      active: {line: 0, character: 0}
+  marks: {}
+finalState:
+  documentContents: |-
+    aaa
+    bbb
+    ccc
+  selections:
+    - anchor: {line: 0, character: 0}
+      active: {line: 0, character: 1}
+    - anchor: {line: 0, character: 1}
+      active: {line: 0, character: 2}
+    - anchor: {line: 0, character: 2}
+      active: {line: 0, character: 3}
+    - anchor: {line: 0, character: 3}
+      active: {line: 1, character: 0}
```

Comment:
For some reason "char" seems to be yielding a newline character here

Reply:
The problem is that `/\p{L}\p{M}*|\P{L}/gu` matches whitespace and new lines as well

Reply:
That is so strange. Why on earth would whitespace be considered a letter or a diacritic?

Reply:
Sounds reasonable. Can we get by just with? `/\p{L}\p{M}*/` or do you have any better suggestions?

Reply:
I think that will miss symbols like brackets. I'd look at the token regex and see what we can steal from there. If you come up blank I can take a look a bit later

Reply:
This seems to work both grapheme and sub token `/\p{L}\p{M}*|[^\p{L}\p{Z}\p{C}]/gu`